app/usecases/sendAbsent.js

```diff
@@ -74,28 +76,25 @@ module.exports = (db, oAuth2Client) => {
 
   this.convertEntityReasons = (waRes) => {
     waRes.entities.forEach((entity) => {
-      if (entity.entity === "absent_reason" ) {
-        this.absentReason= entity.value;
+      if (entity.entity === "absent_reason") {
+        this.absentReason = entity.value;
       }
     });
   };
   // sets Absent Times depending on what was already specified in the Watson Assistant response
-  this.setAbsentTimes =() =>{
+  this.setAbsentTimes = () => {
+    const today = moment().format("YYYY-MM-DD");
```

Comment:
wie mocke ich das in einem test dann?

Reply:
https://medium.com/front-end-weekly/mocking-moment-js-in-jest-a-simple-one-line-solution-61259ffaaa2

Hier haben die das gemockt. Vielleicht hilft das.